Connection termination update (#1157)
ietf-tapswg/api-drafts: draft-ietf-taps-impl.md

# Implementing Connection Termination

> For Close (provoking a Finished event) and Abort (provoking a ConnectionError event), it

Reviewer: ```suggestion
For Close (which leads to a Finished event) and Abort (which leads to a ConnectionError event),
```

> is desirable to be informed when a peer closes or aborts a

Reviewer: ```suggestion
the application might find it useful to be informed when a peer closes or aborts a
```

Connection. Whether this is possible depends on the underlying protocol, and no guarantees

> can be given. When a Connection is mapped to an SCTP stream, the transport system can use the stream reset procedure to cause a Finish event upon a Close action from the peer {{NEAT-flow-mapping}}.

Reviewer: ```suggestion
can be given. For example, when a Connection is mapped to an SCTP stream, the transport system can use the stream reset procedure to cause a Finish event upon a Close action from the peer {{NEAT-flow-mapping}}.
```

> can be given. When an underlying transport connection such as SCTP supports multi-streaming, the transport system can use a stream reset procedure to cause a Finish event upon a Close action from the peer {{NEAT-flow-mapping}}.

Author: ```suggestion
can be given. When an underlying transport connection supports multi-streaming (such as SCTP), the transport system can use a stream reset procedure to cause a Finish event upon a Close action from the peer {{NEAT-flow-mapping}}.
```
